SPECS/hyperv-daemons/hyperv-daemons.spec

```diff
@@ -9,7 +9,7 @@
 Summary:        Hyper-V daemons suite
 Name:           hyperv-daemons
 Version:        5.4.83
-Release:        2%{?dist}
+Release:        3%{?dist}
```

Comment:
In general, it is not required to update the release number for `hyperv-daemons` or `kernel-hyperv` when updating `kernel.spec`.

Reply:
Sounds good. Would it be better to revert this change or keep as-is? Do you have a preference?

Reply:
Please revert so release=2. Less overall delta.

Reply:
Reverted